### ISSUE TITLE
[MAS2.4.6] [Headings and Labels - Open Bot Dialog] Heading should be defined for the pop-up or dialog.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1899]
**Actual Result:**
When the user opens "Open Bot Dialog" button, dialog box opens but the voiceover didn't announces the name of the dialog box. This is because the name i.e. "Open a Bot" is not defined in heading.

**Expected Result:**
The name of the dialog box or pane should be defined under heading tag so that voiceover relevent information.

**Note:**
Same issue observed for each and every dialog box or pop-ups throughout the application.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the "Open Bot" button and activate it.
4. Observe the issue.

**User Impact:**
Users who rely on voiceover will be affected if headings are not defined for screen then the users will be affected to navigate to the screen.

**Test Environment:**
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-80829)

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}](url)

**Attachment:**
[Heading not defined.zip](https://github.com/microsoft/BotFramework-Emulator/files/3686102/Heading.not.defined.zip)

**Comment by ShikhaMishra11** [comment]
Verified the issue and it is fixed in latest build.
OS: MacOs Version 10.15
Application: Bot Framework V4 Emulator (Version 4.5.2-85838)
